src/deterministic_vi/elbo_objective.jl

```diff
@@ -1,4 +1,101 @@
 """
+Convolve the current locations and galaxy shapes with the PSF.  If
+calculate_derivs is true, also calculate derivatives and hessians for
+active sources.
+
+Args:
+ - psf: A vector of PSF components
+ - ea: The current ElboArgs
+ - b: The current band
+ - calculate_derivs: Whether to calculate derivatives for active sources.
+
+Returns:
+ - star_mcs: An array of BvnComponents with indices
+    - PSF component
+    - Source (index within active_sources)
+ - gal_mcs: An array of BvnComponents with indices
+    - PSF component
+    - Galaxy component
+    - Galaxy type
+    - Source (index within active_sources)
+  Hessians are only populated for s in ea.active_sources.
+"""
+function load_bvn_mixtures{NumType <: Number}(
+                    ea::ElboArgs{NumType},
+                    b::Int;
+                    calculate_derivs::Bool=true,
+                    calculate_hessian::Bool=true)
+    # call bvn loader from the Model Module
+    Model.load_bvn_mixtures(ea.S, ea.patches, ea.vp, ea.active_sources,
+                            ea.psf_K, b,
+                            calculate_derivs=calculate_derivs,
+                            calculate_hessian=calculate_hessian)
+end
```

Comment:
I think these came in accidentally from an old version of the file in the merge since I wasn't using them.  So I'll delete them.

I disagree, but more importantly, I think it would be useful to have a set of coding standards that we can all write to.  Right now we are oscillating between one style and another.  These date to the end of the summer, IIRC, when you presumably had a different opinion.